<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Country_Code_Entry">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Country_Entry">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Country_Code_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Country Code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Country_Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Country:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Country_Button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Country</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
